modules/customize-certificates-view-service-serving.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assembly:
+//
+// * authentication/certificates/service-serving-certificate.adoc
+
+[id="view-service-serving-{context}"]
+= View the expiration date of the generated service certificate
+
+While the certificate and key pair is automatically replaced when
+it gets close to expiration, it is possible to view the expiration
+date within the secret.
+
+.Prerequisites
+
+* A secret containing the certificate and key pair must 
```

Comment:
s/must have been generated/must exist

Reply:
Per above, I'm deleting this file and removing the reference.